tests/phpunit/unit-tests/Model/TestModelPDFMetaBoxSuccess.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace GFPDF\Model;
+
+use GFPDF\Controller\Controller_PDF;
+use GFPDF\Helper\Helper_Url_Signer;
+use GFPDF\View\View_PDF;
+use WP_UnitTestCase;
+
+/**
+ * @package     Gravity PDF
+ * @copyright   Copyright (c) 2020, Blue Liquid Designs
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ */
+
+/**
+ * Class TestModelPDFSuccess
+ *
+ * @package GFPDF\Model
+ *
+ * @group   pdf
+ */
+class TestModelPDFMetaBoxSuccess extends WP_UnitTestCase {
```

Comment:
You can combine this with the other file. The idea is to test each iteration in a different method, rather than in the same method like we've been doing. Make sense?